### DISCUSSION TITLE
Создание маршрута от одной точки до другой

### DISCUSSION THREAD

**Discussion by Kirill121195** [yandex-maps-unofficial/vue-yandex-maps#241] [Q&A (2.x)]
Добрый вечер! Во-первых, спасибо за Ваши труды! Сам вопрос, возможно ли как-то рисовать маршрут от точки до точки? Использую Nuxt3

**Comment by daniluk4000** [comment]
Доброго вечера. Спасибо за добрые слова!

У Яндекса есть метод `ymaps3.route`, но он не работает :(

https://yandex.ru/dev/jsapi30/doc/ru/examples/cases/building-route -- вот тут они его используют, но простым смертным он недоступен

Сам маршрут рисовать несложно с помощью полигонов (компонент feature) - вопрос только в том, как получить точки для этого маршрута)  У Яндекса есть API маршрута, но оно, кажется, исключительно платное и стоит от 150 тысяч. Вроде бы вот это: https://yandex.ru/dev/distance_matrix/doc/ru/quickstart

**Reply by Kirill121195** [reply]
Спасибо за обратную связь. Печально, что на бесплатной основе не воспользуешься, значит буду думать)
